IPs/RSKIP185.md

```diff
@@ -0,0 +1,70 @@
+# Title
+
+|RSKIP          |185           |
+| :------------ |:-------------|
+|**Title**      |Peg-out refund and events |
+|**Created**    |19-NOV-20 |
+|**Author**     |JD |
+|**Purpose**    |Usa |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+
+## Abstract
+
+The Bridge offers a protocol to convert R-BTC back in BTC. This functionality is known as a peg-out.
+To do this a user has to send R-BTC to the Bridge and it will start the operation.
```

Comment:
```suggestion
To do this a user has to send R-BTC to the Bridge and it will start the operation. There is a minimum amount of R-BTCs established that the user has to peg-out at once. In the current protocol, if less than that amount is sent to the Bridge those funds are lost.
```